actions/insights/general_setup.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Text, Dict, List
+
+from rasa_sdk import Tracker, FormValidationAction
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk.types import DomainDict
+
+from actions.actions import all_required_slots_are_set
+from actions.slot_match import FuzzySlotMatch, FuzzySlotMatchOption, resolve_slot_match
+from common import logging
+
+
+logger = logging.initialize_logging()
+
+product_kind = FuzzySlotMatch(
+    "insights_product_kind",
+    [
+        FuzzySlotMatchOption("ansible"),
+        FuzzySlotMatchOption("rhel", ["rhel", "redhat", "red-hat", "red hat"]),
```

Comment:
With these two options, we are going to have the same problems as https://github.com/RedHatInsights/astro-virtual-assistant/pull/413 since it matches per word, so "open shift" and "red hat" wouldn't match. (red hat here still doesn't make sense to me though haha)

Reply:
oof... totally forgot about #413 , thanks! Just updated!
The "open shift" one hurts, since I can see our users typing it :disappointed:

Reply:
Yeah, we should make a function for pairs of words sooner or later :smile: